docker-compose.yml

```diff
@@ -1,11 +1,31 @@
 version: '2'
 services:
   app:
+    environment:
```

Comment:
Why does mongo need to be in Docker? It's easy enough to install on all OSs and besides we need to access the Mongolab DBs in production.

Reply:
I work on a lot of different projects with a wide variety of dependencies, so this is a convenient way to keep each set of service dependencies isolated. It's for a standalone dev environment, essentially.

Feel free to revert this and I keep keep an uncommitted version locally to facilitate development.